week-2/extended-project/banzai/todo-list/cli_test.rb

```diff
@@ -0,0 +1,106 @@
+require "./cli"
+
+def init
+  puts "init"
+  @task = read_file
+end
+
+def back
+  puts "back"
+  save_file(@task)
+end
+
+RSpec.describe "Test cli" do
+  before(:each) do
+    init
+  end
+
+  after(:each) do
+    back
+  end
+  
+  # context "in method read" do
+  #   it "expect read_file! return 3 elements" do
+  #     tasks = read_file
+  #     expect(tasks.length).to eq(3)
+  #   end
+  # end
+
+  context "in method show" do
+    it "expect tasks to show" do      
+      tasks = show
+      expect(tasks).to be_a Array
```

Comment:
Could you add more tests here?

Reply:
Sure, we'll add some extra tests to make sure everything works fine.